apps/backend/src/modules/contributors/contributor.repository.ts

```diff
@@ -1,16 +1,25 @@
 import { raw } from "objection";
 
 import { type Repository } from "~/libs/types/types.js";
+import { type GitEmailModel } from "~/modules/git-emails/git-email.model.js";
```

Comment:
```suggestion
import { type GitEmailModel } from "~/modules/git-emails/git-email.js";
```
export the model using the module root file

Reply:
agree

Reply:
done